doc/releases/changelog-0.35.0.md

```diff
@@ -4,25 +4,170 @@
+    tensor([[0.81080498+0.j, 0.        +0.j],
+        [0.        +0.j, 0.18919502+0.j]], requires_grad=True)
+    ```
+    
+  * It is now more intuitive to handle and differentiate parametrized Qiskit circuits. Consider the following circuit:
+
+    ```python
+    from qiskit.circuit import Parameter
+    from pennylane import numpy as np
 
-* A Qiskit `SparsePauliOp` can be converted into a PennyLane `Operator` using `qml.from_qiskit_op`.
+    angle0 = Parameter("x")
+    angle1 = Parameter("y")
+
+    qc = QuantumCircuit(2, 2)
+    qc.rx(angle0, 0)
+    qc.ry(angle1, 1)
+    qc.cx(1, 0)
+    ```
+    
+    We can convert this circuit into a QNode with two arguments, corresponding to `x` and `y`:
+
+    ```python
+    measurements = qml.expval(qml.PauliZ(0))
+    qfunc = qml.from_qiskit(qc, measurements)
+    qnode = qml.QNode(qfunc, dev)
+    ```
+    
+    The QNode can be evaluated and differentiated:
+
+    ```pycon
+    >>> x, y = np.array([0.4, 0.5], requires_grad=True)
+    >>> qnode(x, y)
+    tensor(0.80830707, requires_grad=True)
+    >>> qml.grad(qnode)(x, y)
+    (tensor(-0.34174675, requires_grad=True),
+     tensor(-0.44158016, requires_grad=True))
+    ```
+
+    This shows how easy it is to make a Qiskit circuit differentiable with PennyLane.
+
+* In addition to circuits, it is also possible to convert operators from Qiskit to PennyLane with a new function called
+  `qml.from_qiskit_op`.
   [(#5251)](https://github.com/PennyLaneAI/pennylane/pull/5251)
 
-<h4>Native mid-circuit measurements on default qubit 💡</h4>
+  A Qiskit
+  [SparsePauliOp](https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp) can be
+  converted to a PennyLane operator using `qml.from_qiskit_op`:
+
+  ```pycon
+  >>> from qiskit.quantum_info import SparsePauliOp
+  >>> qiskit_op = SparsePauliOp(["II", "XY"])
+  >>> qiskit_op
+  SparsePauliOp(['II', 'XY'],
+                coeffs=[1.+0.j, 1.+0.j])
+  >>> pl_op = qml.from_qiskit_op(qiskit_op)
+  I(0) + X(1) @ Y(0)
+  ```
+
+  Combined with `qml.from_qiskit`, it becomes easy to quickly calculate quantities like expectation
+  values by converting the whole workflow to PennyLane:
 
-* When operating in finite-shots mode, the `default.qubit` device now performs mid-circuit
-  measurements in a similar way to quantum hardware. For each shot, when a mid-circuit measurement
-  is encountered, the device evaluates the probability of projecting onto `|0>` or `|1>` and
-  makes a random choice to collapse the circuit state.
+  ```python
+  qc = QuantumCircuit(2)  # Create circuit
+  qc.rx(0.785, 0)
+  qc.ry(1.57, 1)
+
+  measurements = [qml.expval(pl_op)]  # Create QNode
+  qfunc = qml.from_qiskit(qc, measurements)
+  qnode = qml.QNode(qfunc, dev)
+  ```
+
+  ```pycon
+  >>> qnode()  # Evaluate!
+  [tensor(0.29317504, requires_grad=True)]
+  ```
+
+<h4>Native mid-circuit measurements on Default Qubit 💡</h4>
+
+* Mid-circuit measurements are now more scalable and efficient in finite-shots mode
```

Comment:
```suggestion
* Mid-circuit measurements can now be more scalable and efficient in finite-shots mode
```
I add this note of caution because deferred measurements can still be much better, e.g., if you have few qubits and many shots.